submissions/Writerman/my_pop_up/css/style.css

```diff
@@ -0,0 +1,239 @@
+  font-family: sans-serif;
+  font-size: 16px;
+  color: #444;
+}
+.nav {
+  padding: 20px;
+  position: absolute;
+  top: 3.0em;
+  right: 2em;
+  width: 350px;
+  border: 2px solid #666;
+  background: #fff;
+}
+.nav_menu {
+  display: grid;
+  column-gap: 48px;
+  grid-template-columns: repeat(4, max-content);
+  justify-content: end;
+}
+.menu_item {
+  display: block;
+  line-height: 1.5em;
+}
+.menu_icon {
+  display: block;
+  width: 1.5em;
+  height: 1.5em;
+  background-size: contain;
+}
+.menu_link {
+  text-decoration: none;
+  color: #666;
+}
+.menu_link:hover >  .icon_user {
+  box-shadow: 10px -10px 22px 10px #000
+}
+.menu_link:focus > .icon_user  {
+  opacity:0.5;
+  box-shadow: 10px -10px 22px 10px #000
+}
+
+.menu_link:hover >  .icon_defend {
+  box-shadow: 10px -10px 22px 10px #000
+}
+.menu_link:focus > .icon_defend  {
+  opacity:0.5;
+  box-shadow: 10px -10px 22px 10px #000
+}
+.menu_link:hover >  .icon_atack{
+  box-shadow: 10px -10px 22px 10px #000
+}
+.menu_link:focus > .icon_atack  {
+  opacity:0.5;
+  box-shadow: 10px -10px 22px 10px #000
+}
+.icon_user {
+  background-image: url(../images/User.png);
+  width: 50px;
+  height: 50px;
+}
+.icon_defend
+{
+  background-image: url(../images/Defend.png);
+  width: 50px;
+  height: 50px;
+}
+.icon_atack {
+  background-image: url(../images/Atack.png);
+  width: 50px;
+  height: 50px;
+}
+.pop_up {
+  display: none;
+    position: absolute;
+    top: 7em;
+    right: 0.6em;
+    width: 350px;
+    border: 2px solid #666;
+    background: #fff;
+}
+.pop_up:before {
+  content: '';
+   display: block;
+   position: absolute;
+   top: -1.1em;
+   right: 32%;
+   width: 2em;
+   height: 2em;
+   border: 2px solid #666;
+   border-right:none;
+   border-bottom: none;
+   background: #fff;
+   transform: rotate(45deg);
+}
+.pop_up_list {
+  display: grid;
+  width: 100%;
+  height: 100%;
+  row-gap: 35px;
+  grid-template-columns: repeat(3, max-content);
+  justify-content: space-between;
+  padding: 40px;
+  overflow-y: auto;
+  box-sizing: border-box;
+}
+.pop_up_list::-webkit-scrollbar {
+  width: 2px;
+  background-color: rgba(102,102,102,0.1);
+}
+.pop_up_list::-webkit-scrollbar-thumb {
+  background-color: rgba(102,102,102,0.5);
+}
+.pop_up_item {
+  display: inline-block;
+}
+.app_link {
+  display: grid;
+  grid-template-rows: repeat(2, max-content);
+  text-decoration: none;
+  color: #666;
+  padding: 3px;
+}
+.app_link:hover,
+.app_link:focus {
+  opacity:0.5;
+  box-shadow: 10px -10px 22px 10px #000
+}
+.app_link:hover .app_name,
+.app_link:focus .app_name {
+  color: #03c;
+  text-decoration: underline;
+}
+.app_link_all_width {
+  grid-column: 1 / -1;
+}
+.app_icon {
+  display: block;
+  width: 50px;
+  height: 50px;
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-position-x: center;
+  background-position-y: bottom;
+  justify-self: center;
+}
+.app_name {
+  display: block;
+  text-align: center;
+}
+.toggle_input {
```

Comment:
Watch the [video](https://www.youtube.com/watch?v=E6kLaaQFctU&t=413s) about hiding checkboxes in the right way